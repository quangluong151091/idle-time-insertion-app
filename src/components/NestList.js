import React from 'react';
import Nest from './Nest';
import Root from './timeline/Root';

class NestList extends React.Component {
    render() {
        return (
            <div>
                <h1 className="List__title">Nest List</h1>
                <table id="table-Nest" style={{ width: '100%' }}>
                    <tbody>
                        <tr className="tblHead">
                            <th className="imgNest">Nest</th>
                            <th className="nameNest">Name</th>
                            <th className="noteNest">Note</th>
                            <th className="sttNest">Status</th>
                        </tr>
                        <Root><Nest img="/pics/nest/29484001PO.png" name="29484001" details="Nest 1"/></Root>
                        <Root><Nest img="/pics/nest/29484002PO.png" name="29484002" details="Nest 2" /></Root>
                    </tbody>
                </table>
            </div>
        );

    }
}

export default NestList