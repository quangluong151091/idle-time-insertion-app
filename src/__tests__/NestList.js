import React from 'react';
import { shallow } from 'enzyme';

import NestList from '../components/NestList';

describe('NestList', () => {
  it('should render correctly', () => {

    const component = shallow(<NestList />);
  
    expect(component).toMatchSnapshot();
  });
});