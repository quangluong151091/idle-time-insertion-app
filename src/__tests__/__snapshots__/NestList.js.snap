// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NestList should render correctly 1`] = `
<div>
  <h1
    className="List__title"
  >
    Nest List
  </h1>
  <table
    id="table-Nest"
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <tbody>
      <tr
        className="tblHead"
      >
        <th
          className="imgNest"
        >
          Nest
        </th>
        <th
          className="nameNest"
        >
          Name
        </th>
        <th
          className="noteNest"
        >
          Note
        </th>
        <th
          className="sttNest"
        >
          Status
        </th>
      </tr>
      <_default>
        <withRouter(Connect(Nest))
          details="Nest 1"
          img="/pics/nest/29484001PO.png"
          name="29484001"
        />
      </_default>
      <_default>
        <withRouter(Connect(Nest))
          details="Nest 2"
          img="/pics/nest/29484002PO.png"
          name="29484002"
        />
      </_default>
    </tbody>
  </table>
</div>
`;
